CHANGELOG.md

```diff
@@ -45,6 +45,11 @@
   inside the closure or `functools.partial` passed as `solver` argument.
 * `nk.optimizer.sr.SRLazyCG` and `nk.optimizer.sr.SRLazyGMRES` have been deprecated and will soon be removed.
 * Parts of the `Lattice` API have been overhauled, with deprecations of several methods in favor of a consistent usage of `Lattice.position` for real-space location of sites and `Lattice.basis_coords` for location of sites in terms of basis vectors. `Lattice.sites` has been added, which provides a sequence of `LatticeSite` objects combining all site properties. Furthermore, `Lattice` now provides lookup of sites from their position via `id_from_position` using a hashing scheme that works across periodic boundaries. [#703](https://github.com/netket/netket/pull/703) [#715](https://github.com/netket/netket/pull/715)
+* The `Grid` API has been retired in favour of specialised `Lattice` constructors ([#724](https://github.com/netket/netket/pull/724)). As a result,
+    * The `color_edges` constructor switch has been removed for the time being. We plan to reverse this later.
+    * `planar_rotation()` and `axis_reflection()` are dropped. Their functionality can be achieved by supplying `point_group()` with the appropriate geometrical point group.
+    * `space_group()` and `lattice_group()` are renamed to the correct terms `point_group()` and `space_group()`. Due to the overlap in the names, deprecation warnings are not provided. Unlike the original `Grid` API, `point_group()` only returns symmetries that leave the origin in place.
```

Comment:
I think the symmetry-related methods were not even part of `v3.0b1` (our first beta version), only one of the `v3.0b1.post#` releases. Not sure if we actually need to add a changelog entry in that case. (Too many entries relating to short-lived features few people have even seen can also be confusing.) @PhilipVinc ?

(This does not apply to the first point about `Grid`, which I think is helpful.)

Reply:
Indeed, those were never released so they shouldn't go in breaking changes. Rather in new features (just one bullet element about all those things, and a short description)

By the way. I don't understand what happened to colored edges...

I am on holiday so I'll properly review tomorrow.

Reply:
> Indeed, those were never released so they shouldn't go in breaking changes. Rather in new features (just one bullet element about all those things, and a short description)

Does this refer to `space_group` and `lattice_group`? Or also `planar_rotations()`?

> By the way. I don't understand what happened to colored edges...

`Lattice` doesn't support them because the only way it takes edges is through KDTree. It will require a more general interface for supplying edges (while keeping KDTree as the default)

Reply:
> Does this refer to space_group and lattice_group? Or also planar_rotations()?

Both.